api/v1/postgres_types.go

```diff
@@ -72,6 +72,8 @@ type PostgresSpec struct {
 	Backup *Backup `json:"backup,omitempty"`
 	// AccessList defines access restrictions
 	AccessList *AccessList `json:"accessList,omitempty"`
+	// SecretName reference to the secret where the user credentials are stored
+	SecretName string `json:"secretname,omitempty"`
```

Comment:
@majst01 
Would `secretName` (json) more consistent? Like in [pod](https://github.com/kubernetes/website/blob/17660852a8e0fa92c6c61a4dd6076092c4f698f7/content/en/examples/pods/inject/secret-pod.yaml#L17)